upgrade/install-3.0.php

```diff
@@ -79,5 +79,15 @@ function upgrade_module_3_0($object, $install = false)
 			) ENGINE='._MYSQL_ENGINE_.' DEFAULT CHARSET=utf8 AUTO_INCREMENT=1');
 	}
 
+	//Add datas to paypal order
+	Db::getInstance()->Execute('
```

Comment:
Hi Jérôme. 
I've added this modifications to the upgrade of 3.0.0 because this columns exists since version 3.0.0

This modifications only need to be applied for people upgrading from 2.8 to a newer version. 
If we add this upgrade to 3.8.0 the upgrade will be applied to everyone, and we might have more problems with people who had no problem.